src/opm/parser/eclipse/EclipseState/Schedule/Schedule.cpp

```diff
@@ -1597,7 +1597,8 @@ void Schedule::iterateScheduleSection(const std::string& input_path, const Parse
                 if (!record.getItem("VOIDAGE_GROUP").defaultApplied(0))
                     voidage_group = record.getItem("VOIDAGE_GROUP").getTrimmedString(0);;
 
-                bool availableForGroupControl = DeckItem::to_bool(record.getItem("FREE").getTrimmedString(0));
+                bool availableForGroupControl = DeckItem::to_bool(record.getItem("FREE").getTrimmedString(0))
+                    && !(group_name == "FIELD");
```

Comment:
That's a peculiar phrasing.  What happened to
```C++
group_name != "FIELD"
```?

Reply:
I'll modify.